src/render.js

```diff
@@ -30,6 +39,15 @@ export default function render(
     getAllByName: getAllByName(instance),
     getAllByText: getAllByText(instance),
     getAllByProps: getAllByProps(instance),
+
+    queryByTestId: queryByTestId(instance),
+    queryByName: queryByName(instance),
+    queryByText: queryByText(instance),
+    queryByProps: queryByProps(instance),
+    queryAllByName: queryAllByName(instance),
+    queryAllByText: queryAllByText(instance),
+    queryAllByProps: queryAllByProps(instance),
+
```

Comment:
Can we additionally expose `root`?
```jsx
root: () => renderer.root;
```

Reply:
Yea, I think that makes sense in this form.

Reply:
After thinking it through, I currently see no real value of exposing this, as we have `get/query` fns to search component tree, and we know the props passed to the root.